tools/apigee-x-trial-provision/apigee-x-trial-provision.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+# shellcheck disable=SC2059,SC2016
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# <http://www.apache.org/licenses/LICENSE-2.0>
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ -z "$PROJECT" ]; then
+   echo "ERROR: Environment variable PROJECT is not set."
+   exit 1
+fi
+
+
+# Step 1: Define functions and environment variables
+function token { echo -n "$(gcloud config config-helper --force-auth-refresh | grep access_token | grep -o -E '[^ ]+$')" ; }
+
+PROJECT_NUMBER=$(gcloud projects describe "$PROJECT" --format="value(projectNumber)")
+export PROJECT_NUMBER
+
+export NETWORK=default
```

Comment:
can we make NETWORK and SUBNET overridable as well?

Reply:
fixed